acinclude/squid-util.m4

```diff
@@ -214,6 +214,17 @@ AC_DEFUN([SQUID_YESNO],[
   AS_IF([test "$1" != "yes" -a "$1" != "no"],[AC_MSG_ERROR([Bad argument for $2: "$1". Expecting "yes", "no", or no argument.])])
 ])
 
+dnl Check that a library is actually available, useable,
+dnl and where its pieces are (eg headers and hack macros)
+dnl Parameters for this macro are:
+dnl 1) binary library name (without 'lib' prefix)
+dnl 2) logic to run checks
```

Comment:
Please rephrase the "logic to run checks". It is difficult to interpret IMO. The PR description has a similar problem. Let's use something more simple/direct than "logic to run" in this context.

I suggest using something along the lines of "code implementing the necessary library checks" or "code implementing the desired library checks" or even just "necessary library checks". I do not insist on any specific wording, but here is one specific suggestion:

```suggestion
dnl 2) necessary library checks (to be executed by this macro unless the use of the library is disabled)
```